frontend/src/app/testQueue/TestCardForm/diseaseSpecificComponents/MultiplexResultInputGroup.tsx

```diff
@@ -0,0 +1,393 @@
+ * COMPONENT
+ */
+interface Props {
+  queueItemId: string;
+  testResults: MultiplexResultInput[];
+  deviceId: string;
+  devicesMap: DevicesMap;
+  onChange: (value: MultiplexResult[]) => void;
+}
+
+const MultiplexResultInputGroup: React.FC<Props> = ({
+  queueItemId,
+  testResults,
+  deviceId,
+  devicesMap,
+  onChange,
+}) => {
+  //eslint-disable-next-line no-restricted-globals
+  const isMobile = screen.width <= 600;
+  const resultsMultiplexFormat: MultiplexResultState =
+    convertFromMultiplexResultInputs(testResults);
+
+  let allResultsInconclusive =
+    resultsMultiplexFormat.covid === TEST_RESULTS.UNDETERMINED &&
+    resultsMultiplexFormat.fluA === TEST_RESULTS.UNDETERMINED &&
+    resultsMultiplexFormat.fluB === TEST_RESULTS.UNDETERMINED;
+
+  const deviceSupportsCovidOnlyResult =
+    doesDeviceSupportMultiplexAndCovidOnlyResult(deviceId, devicesMap);
+  const isFluOnly = isDeviceFluOnly(deviceId, devicesMap);
+
+  if (isFluOnly) {
+    allResultsInconclusive =
+      resultsMultiplexFormat.fluB === TEST_RESULTS.UNDETERMINED &&
+      resultsMultiplexFormat.fluA === TEST_RESULTS.UNDETERMINED;
+  }
+
+  /**
+   * Handle Setting Results
+   */
+  const setMultiplexResultInput = (
+    diseaseName: "covid" | "fluA" | "fluB",
+    value: TestResult
+  ) => {
+    let newResults: MultiplexResultState = resultsMultiplexFormat;
+    if (allResultsInconclusive) {
+      newResults = {
+        covid: TEST_RESULTS.UNKNOWN,
+        fluA: TEST_RESULTS.UNKNOWN,
+        fluB: TEST_RESULTS.UNKNOWN,
+      };
+    }
+    newResults[diseaseName] = value;
+    convertAndSendResults(newResults);
+  };
+
+  const convertAndSendResults = (multiplexResults: MultiplexResultState) => {
+    const results = convertFromMultiplexResult(multiplexResults);
+    onChange(results);
+  };
+
+  /**
+   * Handle Inconclusive
+   */
+
+  const handleInconclusiveSelection = (value: any) => {
+    const markedInconclusive = value.target.checked;
+    if (markedInconclusive) {
+      const inconclusiveState: MultiplexResultState = {
+        covid: isFluOnly ? TEST_RESULTS.UNKNOWN : TEST_RESULTS.UNDETERMINED,
+        fluA: TEST_RESULTS.UNDETERMINED,
+        fluB: TEST_RESULTS.UNDETERMINED,
+      };
+      convertAndSendResults(inconclusiveState);
+    } else {
+      const currentState = { ...resultsMultiplexFormat };
+      if (currentState.covid === TEST_RESULTS.UNDETERMINED) {
+        currentState.covid = TEST_RESULTS.UNKNOWN;
+      }
+      if (currentState.fluA === TEST_RESULTS.UNDETERMINED) {
+        currentState.fluA = TEST_RESULTS.UNKNOWN;
+      }
+      if (currentState.fluB === TEST_RESULTS.UNDETERMINED) {
+        currentState.fluB = TEST_RESULTS.UNKNOWN;
+      }
+      convertAndSendResults(currentState);
+    }
+  };
+
+  const isSomeFluFilled =
+    isFluAFilled(resultsMultiplexFormat) ||
+    isFluBFilled(resultsMultiplexFormat);
+  const areAllResultsRequired =
+    isSomeFluFilled || !deviceSupportsCovidOnlyResult;
+
+  return (
```

Comment:
<img width="495" alt="Screenshot 2023-10-23 at 15 26 27" src="https://github.com/CDCgov/prime-simplereport/assets/20211771/e7a6469c-f9f8-47cf-828e-e9cab9c06457">
Something we can also flag for design is marking the labels as required when we select "Mark test as inconclusive" is misleading

Reply:
that's a good catch 🔍

Reply:
We're likely going to remove that inconclusive checkbox altogether in favor of simplifying the result validation rules for multiplex devices. That change should be in the upcoming PR for supporting HIV on the test card